### ISSUE TITLE
Error on when running "ionic serve" on a new ionic 2 project.

### ISSUE THREAD

**Issue by jQrgen** [driftyco/ionic2#335]
<b>jqrgen@jQrgenpc:~$ ionic start cutePuppyPics --v2</b>
<i>Starting an Ionic v2 application
Downloading: https://github.com/driftyco/ionic2-starter-tabs/archive/master.zip
Installing Node Modules
Updated the hooks directory to have execute permissions
Adding native plugins
+---------------------------------------------------------+
+ New Ionic Updates for October 2015
+
+ The View App just landed. Preview your apps on any device
+ http://view.ionic.io
+
+ Invite anyone to preview and test your app
+ ionic share EMAIL
+
+ Generate splash screens and icons with ionic resource
+ http://ionicframework.com/blog/automating-icons-and-splash-screens/
+
+---------------------------------------------------------+

New! Add push notifications to your Ionic app with Ionic Push (alpha)!
https://apps.ionic.io/signup

New to Ionic? Get started here: http://ionicframework.com/docs/v2/getting-started

Starting an Ionic v2 application
Downloading: https://github.com/driftyco/ionic2-starter-tabs/archive/master.zip
Installing Node Modules
Updated the hooks directory to have execute permissions
Adding native plugins
+---------------------------------------------------------+
+ New Ionic Updates for October 2015
+
+ The View App just landed. Preview your apps on any device
+ http://view.ionic.io
+
+ Invite anyone to preview and test your app
+ ionic share EMAIL
+
+ Generate splash screens and icons with ionic resource
+ http://ionicframework.com/blog/automating-icons-and-splash-screens/
+
+---------------------------------------------------------+

New! Add push notifications to your Ionic app with Ionic Push (alpha)!
https://apps.ionic.io/signup

New to Ionic? Get started here: http://ionicframework.com/docs/v2/getting-started

</i>
<b>jqrgen@jQrgenpc:~/cutePuppyPics$ ionic serve</b><i>
∆ Compiling your SASS to CSS.
√ Sass compiled to CSS.
∆ Running postcss and autoprefixer on compiled CSS.
√ CSS processed to: /home/jqrgen/cutePuppyPics/www/build/css/app.css 
√ Using your webpack.config.js file: /home/jqrgen/cutePuppyPics/webpack.config.js 

√ Webpack complete

∆ Starting browser sync.

[BS] Access URLs:
 --------------------------------------
       Local: http://localhost:8100
    External: http://192.168.5.138:8100
 --------------------------------------
          UI: http://localhost:3001
 UI External: http://192.168.5.138:3001
 --------------------------------------
[BS] Serving files from: www
[BS] Watching files...
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn google chrome ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1046:32)
    at child_process.js:1137:20
    at process._tickCallback (node.js:355:11)
</i>

**Comment by triniwiz** [comment]
I was having the same problem so i used gulp serve instead

**Comment by jQrgen** [comment]
Gulp serve works!

**Comment by sclausen** [comment]
It's a problem in `ionic-app-lib/serve.js`
The browser is hardcoded in `Serve.serveV2` in line 45
``` javascript
    var compilePromise,
        openBrowser = options.launchBrowser,
        browserToOpen = 'google chrome';
```

**Comment by triniwiz** [comment]
I'm still getting this problem :(  and i'm seeing it marked as fixhttps://github.com/driftyco/ionic-app-lib/commit/cbdf611c6af9481a097ccdb3156536b63dc6add8

**Comment by sclausen** [comment]
@triniwiz but the last release is 9 days ago and this commit is 4 days ago, so you can't have any changes regarding this problem. The commit you mentioned is unfortunately not the one fixing this problem. the whole browserToOpen stuff has been removed and much more.

**Comment by jbavari** [comment]
@triniwiz @sclausen What version of the CLI are you all using? 

We previously had browser-sync, which was using the invalid browser name. 

However, we've pulled out Browser-sync to continue using a simple web server and live reload.

If not using the latest CLI version (`npm install -g ionic@alpha`), please try again with the latest CLI (2.0.0-alpha.18) and let us know.

**Comment by sclausen** [comment]
@jbavari The issue I filed has been closed two days ago (driftyco/ionic-app-lib#37) and since then everything was okay.
